.changeset/itchy-donuts-love.md

```diff
@@ -0,0 +1,23 @@
+---
+'@astrojs/netlify': minor
+---
+
+Adds "includedFiles" and "excludedFiles" options. These allow extra files to be deployed in the SSR function bundle.
+
+When an Astro site using `server` or `hybrid` rendering is deployed to Netlify, the generated functions trace the server dependencies and include any that may be needed in SSR. However sometimes you may want to include extra files that are not detected as dependencies, such as files that are loaded using `fs` functions. Also, you may sometimes want to specifically exclude dependencies that are bundled automatically. For example, you may have a Node module that includes a large binary.
+
+The `includedFiles` and `excludedFiles` options allow you specify these inclusions and exclusions as an array of file paths relative to the project root. Both options support glob patterns, so you can include/exclude multiple files at once.
+
+The paths are relative to the site root. If you are loading them using filesystem functions, make sure you resolve paths relative to the site root, and not relative to the source file. At runtime the compiled file will be in a different location, so paths that are relative to the file will not work. You should instead resolve the path using `path.resolve()` or `process.cwd()`, which will give you the site root.
```

Comment:
```suggestion
The `includedFiles` and `excludedFiles` options allow you specify these inclusions and exclusions as an array of file paths relative to the site root. Both options support glob patterns, so you can include/exclude multiple files at once.

If you are specifying files using filesystem functions, resolve the path using `path.resolve()` or `process.cwd()`, which will give you the site root. At runtime, compiled source files will be in a different location and you cannot rely relative file paths.
```

Just noting the above seemed a little repetative (array of paths relative to the site root.. the paths are relative to the site root... resolve paths relative to the site root...) :smile: 

Something like the above maybe conveys the info more compactly, and doesn't make people read more than necessary to get the info they need? (Will note that as a *changeset*, this doesn't have to be full, cover-everything, documentation either! We can spell things out more clearly and exhuastively in docs itself if you think that's necessary!)

Do you think something like this does a good enough "changelog" job?